install.php

```diff
@@ -378,7 +378,7 @@ function store_data($d){
     if ($d['acl']) {
         // hash the password
         $phash = new \dokuwiki\PassHash();
-        $pass = $phash->hash_smd5($d['password']);
+        $pass = $phash->hash_bcrypt($d['password']);
```

Comment:
smd5 is mentioned later another time: https://github.com/splitbrain/dokuwiki/blob/267bbbca31c7b508d47a961a99012cb9c725e1f4/install.php#L398

Reply:
Oops, addressed it, thanks!